### ISSUE TITLE
declare_action_class.class_eval not detected when using libraries

### ISSUE THREAD

**Issue by damacus** [chef/cookstyle#895]
# Version:

7.15.4

# Environment:

GitHub Actions/macOS

# Scenario:

Cookstyle fails to detect `declare_action_class.class_eval` 

See this old Docker file for an example: 

https://github.com/sous-chefs/docker/blob/91d5b1b25c9b004d803aa3cb6cf108fcecd82654/libraries/docker_base.rb#L130-L135

# Steps to Reproduce:

Create a resource under libraries, using the old format

```ruby
module DockerCookbook
  class DockerBase < Chef::Resource
    ...
    declare_action_class.class_eval do
    end
  end
end
```

# Expected Result:

Autocorrection to.  `action_class`

# Actual Result:

Cookstyle doesn't find or correct the old format.

**Comment by tas50** [comment]
First part is here: https://github.com/chef/cookstyle/pull/897

**Comment by tas50** [comment]
Fully resolved now with https://github.com/chef/cookstyle/pull/915
